src/types/macros.test.tsx

```diff
@@ -20,3 +21,40 @@ it("doesn't substitute if no macro matches", (): void => {
   const substituted = resolveMacros("X${F}", macroMap);
   expect(substituted).toEqual("X${F}");
 });
+
+it("varying brackets are still substituted", (): void => {
```

Comment:
I really like this test. Exposes questionable behaviour.

Reply:
Thank you! Really messes with my bracket colourer in vscode though